### DISCUSSION TITLE
How to edit Clip Operator in OnnxRuntime?

### DISCUSSION THREAD

**Discussion by IzanCatalan** [microsoft/onnxruntime#16314] [Other Q&A]
Hi everyone, I am doing inference with an onnx model with Python API onnxruntime-gpu with clip operator. I would like to modify its behaviour by assigning 0 to the input data that exceed the interval instead of limiting the given input within an interval. 

I would like to do so without modifying the internal operator structure, the only operator domain or other related paths affected. 

I also do not want to make changes to the onnx file, where the clip nodes are defined, I just want to change the onnxruntime code where the clip function is compared and assigned when performing the inference.

Is this possible, or must I add a custom clip operator with a new domain to onnxruntime?  How can I archive this?

Thanks.

**Comment by xadupre** [comment]
It should be doable with two operators Where instead of one Clip. What do you think?
